src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -8032,6 +8022,223 @@ public static void F(IExceptionalInterface i)
+            comp2A.VerifyDiagnostics(
+                // (6,19): warning CS8305: 'N.A.B' is for evaluation purposes only and is subject to change or removal in future updates.
+                //         A a = new A.B();
+                Diagnostic(ErrorCode.WRN_Experimental, "A.B").WithArguments("N.A.B").WithLocation(6, 19),
+                // (7,9): warning CS8305: 'N.A.F(params object[])' is for evaluation purposes only and is subject to change or removal in future updates.
+                //         a.F();
+                Diagnostic(ErrorCode.WRN_Experimental, "a.F()").WithArguments("N.A.F(params object[])").WithLocation(7, 9));
+
+            var comp2B = CreateStandardCompilation(source2, new[] { new CSharpCompilationReference(comp1) });
+            comp2B.VerifyDiagnostics(
+                // (6,19): warning CS8305: 'N.A.B' is for evaluation purposes only and is subject to change or removal in future updates.
+                //         A a = new A.B();
+                Diagnostic(ErrorCode.WRN_Experimental, "A.B").WithArguments("N.A.B").WithLocation(6, 19),
+                // (7,9): warning CS8305: 'N.A.F(params object[])' is for evaluation purposes only and is subject to change or removal in future updates.
+                //         a.F();
+                Diagnostic(ErrorCode.WRN_Experimental, "a.F()").WithArguments("N.A.F(params object[])").WithLocation(7, 9));
+        }
+
+        // Unlike [Obsolete] and [Deprecated], there is no
+        // specific error for collection initializers.
+        [Fact]
+        public void TestExperimentalAttributeOnCollectionInitializer()
+        {
+            var source =
+@"using System.Collections;
+using Windows.Foundation.Metadata;
+class C : IEnumerable
+{
+    [Experimental]
+    public void Add(int i)
+    {
+    }
+    public void Add(object o)
+    {
+    }
+    public IEnumerator GetEnumerator() => null;
+    static void M()
+    {
+        var o = new C() { 1, null, 3 };
+    }
+}";
+            var comp = CreateStandardCompilation(new[] { Parse(ExperimentalAttributeSource), Parse(source) });
+            comp.VerifyDiagnostics(
+                // (15,27): warning CS8305: 'C.Add(int)' is for evaluation purposes only and is subject to change or removal in future updates.
+                //         var o = new C() { 1, null, 3 };
+                Diagnostic(ErrorCode.WRN_Experimental, "1").WithArguments("C.Add(int)").WithLocation(15, 27),
+                // (15,36): warning CS8305: 'C.Add(int)' is for evaluation purposes only and is subject to change or removal in future updates.
+                //         var o = new C() { 1, null, 3 };
+                Diagnostic(ErrorCode.WRN_Experimental, "3").WithArguments("C.Add(int)").WithLocation(15, 36));
+        }
+
+        // Combinations of attributes.
+        [Fact]
+        public void TestDeprecatedAndExperimentalAndObsoleteAttributes()
```

Comment:
>TestDeprecatedAndExperimentalAndObsoleteAttributes [](start = 20, length = 50)

I think we are suppressing Obsolete/Deprecated errors inside types/members marked with Obsolete/Deprecated attribute. Should they be suppressed inside types/members marked with Experimental attribute?